src/GovUk.Education.ExploreEducationStatistics.Admin/Security/AuthorizationHandlers/UpdateSpecificMethodologyAuthorizationHandlers.cs

```diff
@@ -7,10 +11,19 @@ public class UpdateSpecificMethodologyRequirement : IAuthorizationRequirement
     }
 
     public class UpdateSpecificMethodologyAuthorizationHandler :
-        HasClaimAuthorizationHandler<UpdateSpecificMethodologyRequirement>
+        AuthorizationHandler<UpdateSpecificMethodologyRequirement, Methodology>
     {
-        public UpdateSpecificMethodologyAuthorizationHandler() : base(SecurityClaimTypes.UpdateAllMethodologies)
+        protected override Task HandleRequirementAsync(
+            AuthorizationHandlerContext context,
+            UpdateSpecificMethodologyRequirement requirement,
+            Methodology methodology)
         {
+            if (methodology.Status == Draft && HasClaim(context.User, SecurityClaimTypes.UpdateAllMethodologies))
```

Comment:
Just wondering if you have tested unapproving a methodology as it looks like this might stop working now. Until we have amendments, analysts need to be able to unapprove them.

In future the approver of the latest release of the publication which the methodology belongs to should be able to unapprove the methodology as long as it's not publicallly accessible yet, so maybe leave a TODO for this here?

Reply:
Will test the unapproving and add the TODO

Reply:
Rolled back behaviour and added a TODO